content/blog/bitigchi/2020-09-11-system-sound-contest.md

```diff
@@ -0,0 +1,64 @@
++++
+type = "blog"
+title = "Contest for System Sounds"
+author = "bitigchi"
+date = "2020-09-06 13:00:00+03:00"
+tags = ["sound", "contest"]
++++
+
+It wouldn't be wrong to say that, Haiku is a unique operating system. It takes its roots from BeOS, the OS everyone envied back in the day, and Haiku follows its footsteps in the most loyal way possible. What could be better than complementing this unique operating system with a unique set of system sounds?
+
+We are looking for contestants (amateur, professional, enthusiast) to produce system sound effects for Haiku. Specifically sounds for:
+
+## Sound Events
+
+**Errors:**
+
+- Beep
+- Error notification
+
+**Notifications:**
+
+- Important notification
+- Information notification
+- Progress notification
+
+**Key/mouse events:**
+
+- Key down
+- Key repeat
+- Key up
+- Keymap switch
+- Mouse down
+- Mouse up
+
+**Window events:**
+
+- Window activated
+- Window close
+- Window minimised
+- Window open
+- Window restored
+- Window zoomed
+
+In addition to these above, a nice soothing **startup chime**. :)
+
+## What to consider
+
+- All sound events except the startup chime should be around/between 1-2 seconds. It **should** not exceed 2 seconds. For startup chime, the limit is 5 seconds.
+- They should be non-intrusive. Distorted tunes are not welcome.
+- Expanding on "haiku" and related themes are encouraged.
+- Entries need to be in WAV format (44,100 Hz, 16 bits ps).
+- Entries should include a note that the work is licensed under MIT or a similar licence.
```

Comment:
https://creativecommons.org/choose/ has a simple license chooser. I think, we should go with _Attribution-ShareAlike 4.0 International_
It's important that we can modify those sounds.

Reply:
Updated.